ipyvolume/widgets.py

```diff
@@ -78,22 +78,30 @@ class Mesh(widgets.Widget):
     color = Array(default_value="red", allow_none=True).tag(sync=True, **color_serialization)
     visible = traitlets.CBool(default_value=True).tag(sync=True)
 
-    material = traitlets.Instance(
-        pythreejs.ShaderMaterial, help='A :any:`pythreejs.ShaderMaterial` that is used for the mesh'
-    ).tag(sync=True, **widgets.widget_serialization)
+    material =  traitlets.Union([
+            traitlets.Instance(pythreejs.ShaderMaterial),
+            traitlets.Instance(pythreejs.MeshPhysicalMaterial),
+            traitlets.Instance(pythreejs.MeshPhongMaterial),
+            traitlets.Instance(pythreejs.MeshLambertMaterial),
+        ], help='A :any:`pythreejs.Material` that is used for the mesh').tag(sync=True, **widgets.widget_serialization)
 
     @traitlets.default('material')
     def _default_material(self):
         return pythreejs.ShaderMaterial(side=pythreejs.enums.Side.DoubleSide)
```

Comment:
```suggestion
        return pythreejs.ShaderMaterial(side=pythreejs.enums.Side.DoubleSide, lights=True)
```
we could this, and remove the `this.lights = true` in the js code (mesh.ts), which makes it slightly more flexible.